config/settings/development.yml

```diff
@@ -27,3 +27,6 @@ web_origin_regex: \Ahttps?:\/\/.*\z
 modules_appeals_api:
   documentation:
     notice_of_disagreements_v1: true
+
+sts:
```

Comment:
Instead of 'development', we use `settings.yml` for config like this. There's currently a block, `mobile_application_platform`, that you could put this in

Reply:
The `settings.yml` for `mvi` has it configured as `mock: false` and then overrides that in `settings/development.yml` with `mock: true`. However, I'm not sure if the second portion of that is necessary, though, since betamocks is disabled in staging via environment variable.

So, I just pushed a change that moves the `mock` setting under the `mobile_application_platform` block as you suggested, but I can also mirror `mvi`'s override if you think it is necessary, though it doesn't look like we need to do that.